pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -85,23 +85,46 @@ all: manager
 # More info on the awk command:
 # http://linuxcommand.org/lc3_adv_awk.php
 
-.PHONY: help
-help:  ## Display this help.
+help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
+##@ Development
+
+manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResource definition objects.
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+
+generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
```

Comment:
```suggestion
generate: controller-gen ## Generate manifests e.g. CRD, RBAC, etc..
```

The description that we have currently shown address better the need. See that it is too verbose which will not fit too much well for the help output. I do not think that we should detail here the files gen.

Reply:
The description I used is to match the one provided in the documentation: https://book.kubebuilder.io/reference/controller-gen.html#generators

Reply:
OK. But there is a doc that has the goal of described detailed what is done.
The output of the tool is just a helper to let users know why/when they should run the command.

Reply:
I feel that the help message for the `generate` target being `Generate code` is the same as having no help message. Same applies for `manifests` target and `Generate manifests` help message.

P.S. it should say `CustomeResourceDefinition` not `CustomResource definition`, fixing that part.